packages/core/test-project/ios/RNEmbraceTests/RNEmbraceTests.swift

```diff
@@ -514,7 +517,44 @@ class EmbraceSpansTests: XCTestCase {
         let exportedSpans = try await getExportedSpans()
         XCTAssertEqual(exportedSpans.count, 2)
    }
+    
+    func testLogNetworkRequest() async throws {
+        module.logNetworkRequest("https://otest.com/v1/products", httpMethod: "get", startInMillis: 1723221815889, endInMillis: 1723221815891, bytesSent: 1000, bytesReceived: 2000, statusCode: 200, error: "", resolver: promise.resolve, rejecter: promise.reject)
 
+        XCTAssertEqual(promise.resolveCalls.count, 1)
+        XCTAssertTrue((promise.resolveCalls[0] as? Bool)!)
+        
+        module.logNetworkRequest("https://otest.com/", httpMethod: "POST", startInMillis: 1723221815889, endInMillis: 1723221815891, bytesSent: 1000, bytesReceived: 2000, statusCode: 200, error: "", resolver: promise.resolve, rejecter: promise.reject)
```

Comment:
would be good to include a test where error is set as well

Reply:
note: no longer valid since we I'm removing the `error` parameter after our last conversation